dist/backend/web-gl/function-node.js

```diff
@@ -889,7 +967,9 @@ module.exports = function (_FunctionNodeBase) {
 		value: function astMemberExpression(mNode, retArr) {
 			debugLog("[in] astMemberExpression " + mNode.object.type);
 			if (mNode.computed) {
-				if (mNode.object.type === 'Identifier' || mNode.object.type === 'MemberExpression' && mNode.object.object.object && mNode.object.object.object.type === 'ThisExpression' && mNode.object.object.property.name === 'constants') {
+				if (mNode.object.type === 'Identifier' || mNode.object.type === 'MemberExpression' &&
+				// mNode.object.object &&
+				mNode.object.object.object && mNode.object.object.object.type === 'ThisExpression' && mNode.object.object.property.name === 'constants') {
```

Comment:
mNode.object.object.object.type -> it looks scary :P (however if it works it is no problem for me :P )

Reply:
>mNode.object.object.object.type -> it looks scary :P (however if it works it is no problem for me :P )

Should be quite a bit easier to read now.  Let me know if that is more acceptable.

Reply:
I wonder why this object is so nested?

Reply:
Noticed i got summoned to this review, might be a little bit too out of touch

But yes I do remember this scary little piece of code, haha. Definitely agree it should be rewritten to be clearer. But im quite sure at that time we were smashing the code together till it works.

```suggestion
if (
	( mNode.object.type === 'Identifier' || mNode.object.type === 'MemberExpression' ) &&
	( mNode.object.object && mNode.object.object.object && 
	  mNode.object.object.object.type === 'ThisExpression' && 
	  mNode.object.object.property.name === 'constants' )
) {
```

Do you think this would be clearer?

Reply:
Btw @robertleeplummerjr - seems like I started having more time on github these days (for various other projects), feel free to drag me in for small reviews if you like.... hopefully I still remember something 😂 : you probably know the library better then I do now

Reply:
Dist files are built.  Can you apply them to src?  Also welcome back!